codegen/src/main/java/org/web3j/codegen/SolidityFunctionWrapper.java

```diff
@@ -98,6 +99,8 @@
     private static final Pattern pattern = Pattern.compile(regex);
     private final GenerationReporter reporter;
 
+    private static Set<String> parentsMethodNames = getParentsMethodNames();
```

Comment:
```suggestion
    private static final Set<String> restrictedMethodNames = new HashSet<>(Arrays.asList(Object.class.getDeclaredMethods()));
```

We only need to build this set once

Reply:
I think suggested code is more better than previous one!